CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-PROJECT(mrpt_vrep_bridge LANGUAGES C CXX)
+PROJECT(mrpt_V-REP_bridge LANGUAGES C CXX)
```

Comment:
Although apparently CMake supports the use of dash in the project name, this makes me super uncomfortable :-)
Plus having it mrpt_vrep_bridge matche the name of the project. Also visually-wise mixing dashes and underscores doesn't look nice.

Reply:
Same here... was tempted to write a note on this but finally desisted, but I totally agree :+1:

Reply:
Done. I changed the name to V-REP without considering this. This wont happen in the future